sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/CodeGeneratorWithInterpretedFallback.scala

```diff
@@ -63,7 +49,10 @@ abstract class CodeGeneratorWithInterpretedFallback[IN, OUT] {
         try {
           createCodeGeneratedObject(in)
         } catch {
-          case CodegenError(_) => createInterpretedObject(in)
+          case _: Exception =>
+            // We should have already see error message in `CodeGenerator`
+            logError("Expr codegen disabled and falls back to the interpreter mode")
```

Comment:
`logWarning` instead of `logError`? It uses `logWarning` in `WholeStageCodegenExec` too.